sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_media.py

```diff
@@ -4,17 +4,163 @@
 # license information.
 # --------------------------------------------------------------------------
 
-from typing import TYPE_CHECKING  # pylint: disable=unused-import
+from typing import Any, List, TYPE_CHECKING  # pylint: disable=unused-import
+from urllib.parse import urlparse
```

Comment:
do we need this library

Reply:
couple other imports as well unsure if needed, such as SDK_MONIKER, TokenCredential ...